### DISCUSSION TITLE
Is Carbon lang will generate .lib and .dll ?

### DISCUSSION THREAD

**Discussion by ns1998-hub** [carbon-language/carbon-lang#1546] [Explorer]
Is Carbon lang will generate .lib and .dll ?

**Comment by jonmeow** [comment]
It's hard to really commit to specific mechanisms, but I'd expect this should be possible for windows compatibility, at least in part. The plan is to be able to use Clang/LLVM's features for Windows/MSVC compatibility, I'm just not quite sure what form that will eventually take.

**Reply by jonmeow** [reply]
To be clear though -- that'll be more a Toolchain thing, not Explorer which is more the demo interpreter and doesn't generate binaries.
